examples/07_advanced/plot_beta_series.py

```diff
@@ -73,7 +73,7 @@
 from nilearn.datasets import fetch_language_localizer_demo_dataset
 from nilearn.glm.first_level import FirstLevelModel, first_level_from_bids
 
-data_dir, _ = fetch_language_localizer_demo_dataset()
+data_dir, _, _ = fetch_language_localizer_demo_dataset()
```

Comment:
I'd avoid doing this unless you plan to deprecate the return behavior in this PR

Reply:
yeah I was not 100% sure that this would justify deprecation

see my comment here: https://github.com/nilearn/nilearn/pull/4215#discussion_r1450382982

but this is not a hill I will die on, so if you feel otherwise we can go through a deprecation

in any case, maybe cleaner to do all of this in a separate PR where we ensure that all data fetchers return a Bunch.

let me know what you think

Reply:
> in any case, maybe cleaner to do all of this in a separate PR where we ensure that all data fetchers return a Bunch.

I would prefer to go with this then

Reply:
okidoki: let me trim this PR down and move the API change to another PR.